.github/workflows/deploy-apidocs.yml

```diff
@@ -11,6 +11,10 @@ on:
       - 'system/**'
       - '.github/workflows/deploy-apidocs.yml'
 
+permissions:
+    contents: write
+    pull-requests: write
```

Comment:
Do we need `pull-requests: write`?

Reply:
Thanks correction, i see deploy with push content only by @MGatner or @kenjis. `pull-requests: write` is mistake.